contentcuration/contentcuration/serializers.py

```diff
@@ -543,10 +544,11 @@ class Meta:
         list_serializer_class = CustomListSerializer
         model = ContentNode
         fields = ('title', 'changed', 'id', 'description', 'sort_order', 'author', 'copyright_holder', 'license',
-                  'license_description', 'assessment_items', 'files', 'parent_title', 'ancestors',
+                  'license_description', 'assessment_items', 'files', 'parent_title', 'content_id',
```

Comment:
If I understand this correctly, we don't export `ancestors` anymore. What changed?